include/wx/generic/private/grid.h

```diff
@@ -283,8 +328,9 @@ class wxGridHeaderCtrl : public wxHeaderCtrl
         GetOwner()->DoEndMoveCol(event.GetNewOrder());
     }
 
-    wxVector<wxGridHeaderColumn> m_columns;
-
+    static wxVector<wxGridHeaderColumn> ms_columns;
```

Comment:
Sorry, but I have a huge problem with this static member -- this can't possibly work as soon as there is more than one `wxGrid` in the application.

Could you please explain why does it have to be static so that we could find another solution to whichever problem this solves? This is a rather fundamental question because it affects the changes done to `wxHeaderCtrl` too, so I'd really like to deal with it first, as we know that we *have* to deal with it somehow.

Thanks!

Reply:
You are right, I did not take into consideration that you would have multiple grids available at the same time.
I needed the columns vector to be the same for the frozen and normal grid labels.
I will try to change it.